dipy/viz/fvtk.py

```diff
@@ -79,6 +79,9 @@
     msg = "Python VTK is not installed"
     warn(msg)
 
+    ren, have_ren, _ = optional_package('dipy.viz.window.ren',
```

Comment:
@arokem 
```
ren, have_ren, _ = optional_package('dipy.viz.window.ren',
                                        'Python VTK is not installed')
```
returns a TripWire object into `ren`

So calling `ren` will only raise an exception not the line 82.